src/guides/v2.3/comp-mgr/cli/upgrade-with-script.md

```diff
@@ -64,6 +64,10 @@ rm -rf <Magento install dir>/var/page_cache/*
 rm -rf <Magento install dir>/generated/code/*
 ```
 
+```bash
+rm -rf <Magento install dir>/generated/metadata/*
```

Comment:
On a fully compiled and deployed site, the only folders in generated are - code & metadata.
Why not simply update the existing information to -
> rm -rf <Magento install dir>/generated/*

Reply:
It will remove the .htaccess file in /generated directory